src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1915,7 +1924,7 @@ public async Task<string> InvokeToStringAsync(IEnumerable<int> typeIds, bool isV
                     foreach (var methodId in methodIds)
                     {
                         var methodInfoFromRuntime = await GetMethodInfo(methodId, token);
-                        if (methodInfoFromRuntime.Info.GetParametersInfo().Length > 0)
+                        if (methodInfoFromRuntime != null && methodInfoFromRuntime.Info.GetParametersInfo().Length > 0)
```

Comment:
If `GetMethodInfo` returned null then it means we don't have access to the assembly with the method. How are we then able to call `InvokeMethod` on it? Shouldn't the condition be like:
`if (methodInfoFromRuntime == null || methodInfoFromRuntime.Info.GetParametersInfo().Length > 0)`?

Reply:
I'm assuming that if we don't have the methodInfoFromRuntime we can try to invoke the ToString believing that it doesn't expect any parameter.